### DISCUSSION TITLE
Can/should riverpod/hooks be used for Flutter Web instead of an actual database for managing data?

### DISCUSSION THREAD

**Discussion by jacob-emery** [rrousselGit/riverpod#1172] [Q&A] [answered]
Hello everyone!
Many databases don't offer support for the web (yet). On top of that, I think that local data storage for the web isn't really needed and I also read that local databases aren't ideal for the web. Can the data that I usually store using a normal database (on Android and iOS) be cached using Riverpod/hooks (without storing it into a database) when using Flutter Web? If not, what is your strategy?

**Comment by rrousselGit** [comment, marked as answer]
Riverpod has no offline storage built-in, so you would still need a database
